lib/SIL/IR/SILFunctionType.cpp

```diff
@@ -1252,6 +1252,49 @@ enum class ConventionsKind : uint8_t {
   CXXMethod = 8,
 };
 
+template <typename T>
```

Comment:
This helper function should be put into a header file, so that it can be used in both `lib/SIL/IR/SILFunctionType.cpp` and `lib/ClangImporter/ImportDecl.cpp`. (Perhaps it can go in `ClangImporter.h`, in the `importer` namespace?)